en/docs/references/concepts/authorization/user-managed-access.md

```diff
@@ -1,48 +1,48 @@
 # User Managed Access (UMA 2.0)
 
-## What is user managed access?
+## What is UMA 2.0?
 
-UMA 2.0 (User Managed Access) is a federated authorization standard protocol approved by the [Kantara Initiative](https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-grant-2.0.html). It is built on top of OAuth 2.0, which enables party-to-party sharing. 
+UMA 2.0 (User Managed Access) is a federated authorization standard protocol approved by the [Kantara Initiative](https://kantarainitiative.org/). It is built on top of OAuth 2.0 and enables party-to-party sharing. 
 
-UMA enables users to share and control access to their protected resources with other parties that request access to them. Additionally, it also strengthens data privacy and helps to comply with modern-day privacy regulations such as [GDPR](TODO:insert-link) and [CCPA](TODO:insert-link). For instance, consider a patient who wishes to share their health data (such as medical prescriptions and lab reports) with their doctor, family members, and health insurance company. They may wish to selectively share some health-related data with certain parties, but may not want to share everything with everyone, and may wish to give them permission to `view` the data, but not `edit` it. UMA 2.0 helps to effectively share this data as well as maintain control over the resource sharing in a secure and reliable manner. 
+UMA enables users to share and control access to their protected resources with other parties that request access to them. Additionally, it also strengthens data privacy and helps to comply with modern-day privacy regulations such as [GDPR](../../compliance/gdpr) and [CCPA](../../compliance/ccpa). For instance, consider a patient who wishes to share their health data (such as medical prescriptions and lab reports) with their doctor, family members, and health insurance company. They may wish to selectively share some health-related data with certain parties, but may not want to share everything with everyone, and may wish to give them permission to `view` the data, but not `edit` it. UMA 2.0 helps to effectively share this data as well as maintain control over the resource sharing in a secure and reliable manner. 
 
-This is done using authorization policies. The speciality of this protocol is that resource owners need not be present online at the time of access, because the cross-party sharing is driven by pre-defined policies.
+This is done using authorization policies. The specialty of this protocol is that resource owners need not be present online at the time of access, because the cross-party sharing is driven by pre-defined policies.
```

Comment:
```suggestion
This is done using authorization policies. The specialty of this protocol is that resource owners need not be present online at the time of access, because the cross-party sharing is driven by predefined policies.
```